integrations/telegram/src/const.ts

```diff
@@ -1,3 +1,5 @@
 export const USER_PICTURE_MAX_SIZE_BYTES = 25_000
 export const INTEGRATION_NAME = 'telegram'
 export const idTag = `${INTEGRATION_NAME}:id` as const
+export const chatIdTag = `${INTEGRATION_NAME}:chatId` as const // Conversation the message belongs to (see: https://core.telegram.org/bots/api#chat)
+export const senderIdTag = `${INTEGRATION_NAME}:senderId` as const
```

Comment:
last minute thought: We should probably rename this to `telegramUserIdTag` and `${INTEGRATION_NAME}:telegramUserId` respectively to make it super clear what this tag contains just by reading the name. This is important because an integration's tags are auto-documented on the hub, for example: https://app.botpress.dev/hub/integrations/238f88e8-107e-4f78-92ed-affe32ef6ad7

Same for `chatId`, `telegramChatId` is probably better.

Reply:
I don't agree with this, since the integration name prefixes the actual tag, there's no need to change it IMO

Reply:
Yes, sorry what I meant to say it's that the provider’s own term for the tag names, so that for example if telegram calls it a “user” we use “user” for the tag rather than “sender”.

Reply:
I renamed the senderId to fromUserId tag